sodium/sodium.php

```diff
@@ -1129,6 +1132,9 @@ function sodium_crypto_secretstream_xchacha20poly1305_keygen(): string {}
  */
 function sodium_crypto_secretstream_xchacha20poly1305_init_push(string $key): array {}
 
+#[PhpStormStubsElementAvailable('8.2')]
```

Comment:
That's an [old function](https://www.php.net/manual/en/function.sodium-crypto-secretstream-xchacha20poly1305-push.php), it's `@since 7.2`